packages/keystone/src/lib/core/types-for-lists.ts

```diff
@@ -152,7 +233,13 @@ export function initialiseLists(
         const { fields } = lists[listKey];
         return Object.fromEntries(
           Object.entries(fields).flatMap(([key, field]) => {
-            if (!field.input?.orderBy?.arg || field.access.read === false) return [];
+            if (
+              !field.input?.orderBy?.arg ||
+              !field.graphql.isEnabled.read ||
+              !field.graphql.isEnabled.orderBy
```

Comment:
I'm not sure about whether `field.graphql.isEnabled.read === false` should disable filtering and ordering, I can see arguments both ways but we should discuss